engines/router/missionctl/server/upi/server.go

```diff
@@ -16,21 +17,17 @@ import (
 	"github.com/caraml-dev/turing/engines/router/missionctl/log/resultlog"
 	"github.com/caraml-dev/turing/engines/router/missionctl/server/constant"
 	"github.com/caraml-dev/turing/engines/router/missionctl/turingctx"
-	upiv1 "github.com/caraml-dev/universal-prediction-interface/gen/go/grpc/caraml/upi/v1"
 	"github.com/gojek/fiber"
 	fibergrpc "github.com/gojek/fiber/grpc"
 	"github.com/opentracing/opentracing-go"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/metadata"
 	"google.golang.org/grpc/reflection"
-	"google.golang.org/protobuf/proto"
 )
 
 const tracingComponentID = "grpc_handler"
 
 type Server struct {
-	upiv1.UnimplementedUniversalPredictionServiceServer
```

Comment:
this is the recommended way to create a server for forward compatibility. But since the server is using `PredictValues(context.Context, []byte) ([]byte, error)` instead, this is no longer useful.